src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -658,13 +679,29 @@ def docInfoUpdate(self, msOutDoc, pipeLine=None):
         """
 
         # Fill the destinationOutputMap first
-        if msOutDoc['isRelVal']:
-            destinationOutputMap = []
-            wflowDstSet = set()
-            updateDict = {}
-            for dataset in msOutDoc['OutputDatasets']:
-                _, dsn, procString, dataTier = dataset.split('/')
-                destination = set()
+        destinationOutputMap = []
+        wflowDstSet = set()
+        updateDict = {}
+        for dataset in msOutDoc['OutputDatasets']:
+            _, dsn, procString, dataTier = dataset.split('/')
```

Comment:
```suggestion
            dataTier = dataset.split('/')[-1]
```

Reply:
done

Reply:
Actually this immediately introduced a BUG, because `dsn` and `procString` are used few lines bellow :). Here is the pylint report: [1] I am reverting this one.

[1]
```
E0602, line 703 in MSOutput.docInfoUpdate: Undefined variable 'dsn'
E0602, line 703 in MSOutput.docInfoUpdate: Undefined variable 'procString'
E0602, line 705 in MSOutput.docInfoUpdate: Undefined variable 'dsn'
E0602, line 705 in MSOutput.docInfoUpdate: Undefined variable 'procString' 
```

Reply:
You're right, sorry!